dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/webapp/WebApp.java

```diff
@@ -4425,6 +4424,17 @@ public void sendError(HttpServletRequest req, HttpServletResponse res, ServletEr
 
             reportRecursiveError(req, res, error, new WebAppErrorReport(th));
         }
+        finally {
+            //since Servlet 6.1 
+            if (WebContainer.isServlet60orAbove) { // update this to isServlet61orAbove when that PR is merged !!!!! and remove this commend
```

Comment:
Need to update this to the correct isServlet61orAbove() once the PR for that code is merged

https://github.com/OpenLiberty/open-liberty/pull/27494/files#diff-68087dbbbc3a7c277754cc940de71aa2b0316d2c26f42f302a55ee2680d5432eR1657

Reply:
@pmd1nh  this has been merged now so let's make this change now :)